core/src/mindustry/ui/dialogs/ModsDialog.java

```diff
@@ -35,6 +36,27 @@ public ModsDialog(){
         addCloseButton();
 
         buttons.button("@mods.guide", Icon.link, () -> Core.app.openURI(modGuideURL)).size(210, 64f);
+        
+        if(Vars.mobile = false) {
+            buttons.button("@mods.reload", Icon.refresh, () -> {
+            
+                try {
+            
+                String jarPath = getClass().getProtectionDomain().getCodeSource().getLocation().toURI().getPath().replaceFirst("/", "").replaceAll("/", "\\" + File.separator).toString();
+            
+                Runtime rt = Runtime.getRuntime();
+                String[] commandAndArguments = {"java","-jar", jarPath};
+                Process p = rt.exec(commandAndArguments);
+            
+                } catch (IOException i) {
+                    i.printStackTrace();
+                } catch (URISyntaxException u) {
+                    u.printStackTrace();
+                };
```

Comment:
handle any error (i.e `Throwable`), then you don't need to import `URISyntaxException`.